python/ray/_private/services.py

```diff
@@ -1469,6 +1469,7 @@ def start_raylet(redis_address,
         sys.executable,
         setup_worker_path,
         f"--worker-setup-hook={worker_setup_hook}",
+        f"--session-dir={session_dir}",
```

Comment:
what is a session in this context?

Reply:
It should mean the same thing as usual.  This is just to pipe the session dir to the shim process, which I think has no other way of knowing it (worker hasn't been started yet so can't use anything like worker.get_session_dir_path)